pub-critic/src/app/components/LikeAndDislikeRow/LikeAndDislike.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
+
+import {
+  checkLikeStatus,
+  toggleDislike,
+  toggleLike,
+} from "api/LikesAndDislikesApi";
+import Image from "next/image";
+import { useEffect, useState } from "react";
+import Like from "assets/Like.svg";
+import Dislike from "assets/Dislike.svg";
+import classes from "./LikeAndDislike.module.scss";
+import clsx from "clsx";
+import { set } from "react-hook-form";
+
+export interface LikeAndDislikeProps {
+  reviewId: number;
+  likeScore: number;
+}
+
+export const LikeAndDislike = ({
+  reviewId,
+  likeScore,
+}: LikeAndDislikeProps) => {
+  const [isLiked, setIsLiked] = useState<boolean>(false);
+  const [isDisliked, setIsDisliked] = useState<boolean>(false);
+  const [likes, setLikes] = useState<number>(likeScore);
+  const [status, setStatus] = useState<number>(0);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    checkStatus();
+  }, []);
+
+  useEffect(() => {
+    checkStatus();
+  }, [reviewId]);
```

Comment:
This will be called twice initially?

Reply:
most likely, will remove it so I do not spam calls

Reply:
:white_check_mark: